FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,14 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+			switch result {
+			case let .success((data, response)):
+				completion(FeedImagesMapper.map(data, response))
+			case .failure:
+				completion(.failure(.connectivity as Error))
```

Comment:
`Error.connectivity` is easier to follow/understand than casting

```suggestion
				completion(.failure(Error.connectivity))
```